### PR TITLE
Replace strings with constants

### PR DESCRIPTION
**Pull request by nmadruga**
Branched from #3 

Just adds functions to check for labels to be using constants instead of strings.

### CODE REVIEW

File: constants/labels.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+const featureTypes = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+function getPrefix (label) {
+    const labelPrefix= label.split(':');
+    return (labelPrefix.length > 0) ? labelPrefix[0].trimRight() : '';
```
Comment (mark-prins) [inline constants/labels.js]: Not strictly necessary as the `split` operator will return an array with one element if the separator character is not found i.e. it always returns an array. The thing to check for is that label is not null though - the split will have a problem with that.

Suggest:

`const getPrefix = (label) => (label || '').split(':')[0].trimRight();`

File: constants/labels.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+function getType (label) {
+    const labelType = label.split(':');
```
Comment (mark-prins) [inline constants/labels.js]: perhaps: 
```suggestion
    const labelType = (label || '').split(':');
```

File: constants/labels.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    switch(prefix) {
+        case prefixes.FEATURE:
+            switch (type) {
+                case featureTypes.EXISTING:
+                    return `\n\n## Improvements`;
+                case featureTypes.NEW:
+                    return `\n\n## New Features`;
+                default:
+
+                    console.log(type, featureTypes.EXISTING, featureTypes.NEW, type === featureTypes.EXISTING, type === featureTypes.NEW);
```
Comment (mark-prins) [inline constants/labels.js]: should the `console.log` still be here?

File: constants/labels.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    return `\n\n## ${prefix}:${type} - No title preset`;
+            }
+        case prefixes.BUG:
+            switch (type) {
+                case bugTypes.PRODUCTION:
+                    return `\n\n## Bug fixes`;
+                default:
+                    console.log(type, bugTypes.PRODUCTION, type === bugTypes.PRODUCTION);
```
Comment (mark-prins) [inline constants/labels.js]: should the `console.log` still be here?

**Review by mark-prins — APPROVED** [review]
Nice! I have some suggestions, but no show stoppers

File: constants/labels.js (hunk 1 above)
Comment (nmadruga) [reply constants/labels.js]: Yes, indeed. It looks way better and simpler like that. Thanks!

**Comment by wlthomson** [comment]
Merging into base branch for review.

**Pull request merged**